File: 0009-palindrome-number/0009-palindrome-number.c
bool isPalindrome(int x) {
    if(x<0){
        return false;
    }
    double reverse=0;
    for(int i=x;i>0;i/=10){
        reverse=reverse*10+(i%10);
    }
    printf("%d",reverse);
    if(reverse==x){
        return true;
    }
    return false;
}

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        if (x<0)return false;
        int reverse=0;
        int val=x;
        while(val!=0){
            reverse=reverse*10+(val%10);
            val/=10;
        }
        return reverse == x ? true : false;
    }
}